orchagent/crmorch.cpp

```diff
@@ -768,7 +768,7 @@ void CrmOrch::checkCrmThresholds()
 
                 res.exceededLogCounter++;
             }
-            else if ((utilization <= res.lowThreshold) && (res.exceededLogCounter > 0))
+            else if ((utilization < res.lowThreshold) && (res.exceededLogCounter > 0))
```

Comment:
This might break other tests which would expect a log when low threshold is reached. Instead can we have the condition as below
utilization is <= res.lowThreshold and  res.lowThreshold != res.highThreshold
The above will handle both scenarios.

Please also make sure you add UT. swss needs coverage.

Reply:
Sure I'll do that. I think CRM is fully covered.

https://github.com/alexrallen/sonic-swss/blob/0526e050226143f9c31d8193542dc61ace7e6b85/tests/test_crm.py

Reply:
The proposed logic will cause the system to never reset if the two thresholds are the same which I don't think is the desired behavior.